concent_api/middleman/middleman_server.py

```diff
@@ -44,6 +47,7 @@ def __init__(
         self._request_queue: asyncio.Queue = asyncio.Queue(loop=self._loop)
         self._response_queue_pool = QueuePool(loop=self._loop)
         self._message_tracker: OrderedDict = OrderedDict()
+        self._ss_connection_candidates: List[Tuple[asyncio.Task, asyncio.StreamWriter]] = []
```

Comment:
I think we should use full name here, `ss` => `signing_service`. Especially in this case ;)

Reply:
quite frankly, I would love to keep it that way - as a form of desperate protest against changes in naming in python community ;)